tests/system/python/e2e/test_e2e_csv_multi_filter_pi.py

```diff
@@ -157,6 +170,70 @@ def test_end_to_end(self, start_south_north, disable_schedule, foglamp_url, read
         if not skip_verify_north_interface:
             self._verify_egress(read_data_from_pi, pi_host, pi_admin, pi_passwd, pi_db, wait_time, retries)
 
+        tracking_details = self.get_asset_tracking_details(foglamp_url, "Ingest")
+        assert len(tracking_details["track"]), "Failed to track Ingest event"
+        tracked_item = tracking_details["track"][0]
+        assert "playfilter" == tracked_item["service"]
+        assert "e2e_filters_RMS" == tracked_item["asset"]
+        assert "playback" == tracked_item["plugin"]
+
+        tracking_details = self.get_asset_tracking_details(foglamp_url, "Filter")
+        assert len(tracking_details["track"]), "Failed to track Filter event"
+        tracked_item = tracking_details["track"][0]
+        assert "playfilter" == tracked_item["service"]
+        assert "e2e_csv_filter_pi" == tracked_item["asset"]
+        assert "fscale" == tracked_item["plugin"]
+
+        tracking_details = self.get_asset_tracking_details(foglamp_url, "Filter")
```

Comment:
Why do we need to hit curl request again? Don't we have all filter event entries on L#180 step?